neqo-transport/src/connection/mod.rs

```diff
@@ -629,13 +629,17 @@ impl Connection {
             .unwrap()
     }
 
+    fn confirmed(&self) -> bool {
```

Comment:
This can't be `const` as well?

Reply:
No, because the derived equality function isn't `const`. There are some Nightly-only features one can set, but they aren't stable.